mobile/src/people/company-departments-level-people.tsx

```diff
@@ -46,6 +41,20 @@ export class CompanyDepartmentsLevelPeople extends Component<CompanyDepartmentsL
         );
     }
 
+    //----------------------------------------------------------------------------
+    private getEmployeesToRender(): Employee[] {
+        const { employees, superiorChiefIds } = this.props;
+
+        if (!superiorChiefIds) {
+            return employees;
+        }
+
+        let employeesToRender = employees.slice().sort(employeesAZComparer);
```

Comment:
Why is it sorted only in this branch?

Reply:
fdaa61f70988aa0f0a54a1c2121eb0e4293e658d